observability/logging/cluster-logging-deploying.adoc

```diff
@@ -7,57 +7,94 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-You can deploy {logging} by installing the {clo}. The {clo} creates and manages the components of the logging stack.
+You deploy {logging} by installing the {loki-op} or {es-op} to manage your log store, followed by the {clo} to manage the components of logging. You can use either the {product-title} web console or the {product-title} CLI to install or configure {logging}.
 
-include::snippets/logging-compatibility-snip.adoc[]
-
-[IMPORTANT]
+[TIP]
 ====
-For new installations, use Vector and LokiStack. Elasticsearch and Fluentd are deprecated and are planned to be removed in a future release.
+The {clo} must be installed *after* the log store Operator.
 ====
 
-ifdef::openshift-origin[]
-[id="prerequisites_cluster-logging-deploying"]
-== Prerequisites
-* Ensure that you have downloaded the {cluster-manager-url-pull} as shown in _Obtaining the installation program_ in the installation documentation for your platform.
-+
-If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in _Configuring {product-title} to use Red Hat Operators_.
-endif::[]
+== With the Elasticsearch log store
 
-//Installing the CLO via webconsole
-include::modules/cluster-logging-deploy-console.adoc[leveloffset=+1]
-include::modules/create-cluster-logging-cr-console.adoc[leveloffset=+1]
+include::snippets/logging-elastic-dep-snip.adoc[leveloffset=+1]
 
-// Install using CLI
-include::modules/cluster-logging-deploy-cli.adoc[leveloffset=+1]
-include::modules/create-cluster-logging-cli.adoc[leveloffset=+1]
+include::modules/logging-es-deploy-console.adoc[leveloffset=+2]
 
-[id="cluster-logging-deploying-postinstallation"]
-== Postinstallation tasks
+include::modules/logging-es-deploy-cli.adoc[leveloffset=+2]
 
-After you have installed the {clo}, you can configure your deployment by creating and modifying a `ClusterLogging` custom resource (CR).
+== With the Loki log store
 
-[TIP]
+.Prerequisites
+
+include::snippets/logging-retention-period-snip.adoc[leveloffset=+1]
```

Comment:
![image](https://github.com/openshift/openshift-docs/assets/77501157/1f2a7e90-f716-4c54-a450-21591be9bc65)

Reply:
I think that the rendering of the .Prerequisites is getting thrown off by the snippet. Maybe put the .Prerequisites after the snippet inclusion?